components/ResilienceResource/CovidBanner.module.css

```diff
@@ -0,0 +1,16 @@
+.CovidBanner {
+  background-color: #95A9EF;
+  padding: 10px 20px;
+}
+
+.CovidBanner h1 {
+  text-transform: uppercase;
+  font-weight: bold;
+  margin: 0px 60px;
+  color: white;     
+}
+
+.CovidBanner p {
+  color: white;  
+  text-align: center;
```

Comment:
might look better to not center the text. I'll ask my sister in the future about it.

Reply:
Do you wanna create an issue by referencing this comment so you don't forget? @lemonface0309

Reply:
Click the 3 dots and there is an option called "reference in new issue"

Reply:
ah ok. I just followed the Figma